src/nvim/generators/gen_api_dispatch.lua

```diff
@@ -164,6 +168,15 @@ local dump_bin_array = require("generators.dump_bin_array")
 dump_bin_array(funcs_metadata_output, 'funcs_metadata', packed)
 funcs_metadata_output:close()
 
+
+local api_doc_metadata_output = io.open(api_doc_metadata_outpuf, 'wb')
+local api_mpack = io.open(arg[7], 'r')
+local api_packed = mpack.pack(api_mpack:read('*all'))
```

Comment:
I could do `packed = ...` here but I changed it to `api_packed` because I wasn't sure if reusing the `packed` variable should be avoided?

Reply:
a different variable is fine